__tests__/unit/repositories/task/create.test.ts

```diff
@@ -0,0 +1,77 @@
+import { DateTime } from 'luxon';
+
+import { taskSchema } from '../../../../src/models/task.model';
+import {
+  CreateTaskDto,
+  TaskPriority,
+  TaskRepository
+} from '../../../../src/repositories/task/create';
+
+describe('create task => TaskRepository.create', () => {
+  it('should save a new task with the correct fields', async () => {
+    const taskRepo = new TaskRepository(taskSchema);
+    const taskDto: CreateTaskDto = {
+      title: 'Learn Go',
+      description: 'Learn web development with Golang'
+    };
+
+    const newTask = await taskRepo.createTask(taskDto);
+    expect(newTask).not.toBe(null);
+
+    const expectedObject = {
+      id: expect.any(String),
+      title: taskDto.title,
+      description: taskDto.description,
+      dateCreated: expect.any(Date),
+      dateUpdated: expect.any(Date)
+    };
+    expect(newTask).toEqual(expect.objectContaining(expectedObject));
+
+    expect(newTask).not.toEqual(
+      expect.objectContaining({
+        _id: expect.anything(),
+        startTime: expect.anything(),
+        createdAt: expect.anything(),
+        updatedAt: expect.anything(),
+        deletedAt: expect.anything(),
+        completionTime: expect.anything(),
+        priorityLevel: expect.anything(),
+        notificationSubscription: expect.anything()
+      })
+    );
```

Comment:
Why?

Reply:
Just wanted to assert that those fields are not present. But given the suggested use of "toStrictEqual" (didn't know about this) above, they will now be discarded.